packages/parser/src/pipeline/classic/segmentToCombat.ts

```diff
@@ -7,11 +7,11 @@ import { CombatUnitReaction, CombatUnitType, ICombatEventSegment } from '../../t
 import { computeCanonicalHash, nullthrows } from '../../utils';
 import { isNonNull } from '../common/utils';
 
-export const segmentToCombat = () => {
+export const segmentToCombat = (timezone: string) => {
   return pipe(
     map((segment: ICombatEventSegment): IArenaMatch | IMalformedCombatData | null => {
       if (segment.events.length >= 3) {
-        const combat = new CombatData('classic');
+        const combat = new CombatData('classic', timezone);
```

Comment:
can this be derived from segment.events so this function doesn't need to take the parameter?

Reply:
Sort of -- to factor it out we effectively need to attach it to every event which might be fine